src/data/nat/modeq.lean

```diff
@@ -124,9 +124,6 @@ end)
 
 end modeq
 
-lemma add_mod (a b n : ℕ) : (a + b) % n = ((a % n) + (b % n)) % n :=
-(nat.modeq.modeq_add (nat.modeq.mod_modeq a n) (nat.modeq.mod_modeq b n)).symm
-
```

Comment:
Should some of the other lemmas be moved to `data.nat.basic` as well?

Reply:
Moving them would likely complicate some of the proofs to avoid circular dependencies. I'm not sure if that's desirable, in the absence of something with a use for those lemmas that can't make use of data.nat.modeq without introducing such dependencies.

Reply:
OK, I'm a bit concerned that `add_mod` and `mul_mod` are separated, but I can certainly believe that things get messy if it gets moved.

Reply:
Concretely, the `modeq` proofs use a lot of theorems from `data.int.basic`, which indirectly imports `data.nat.basic`.

Reply:
How hard would it be to move `mul_mod` next to `add_mod` again (and change the proof to something a bit more low-level)? I would hope that a proof similar to the one you now have for `add_mod` would work.

Reply:
Moved `mul_mod` as well.